.github/workflows/flake-detector.yml

```diff
@@ -0,0 +1,41 @@
+name: Flake detector
+on:
+  # Enable triggering the workflow manually
+  workflow_dispatch:
+  push:
+    branches: [ "master" ]
+
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
+    timeout-minutes: 75
```

Comment:
Will it run on the each PR? If yes isn't it more time to detect failure. Our development cycle may get affect due to that.

Reply:
+1. 
Can you add more details to the PR. It runs 5 times and does it report even though the test failed once?

Reply:
It'll be run as a postsubmit i.e. after the code gets pushed to master, so won't block submissions. 
And it reports an error for each time a particular test failed.